src/components/reservation.css

```diff
@@ -41,7 +44,23 @@
 }
 
 @media (max-width: 600px) {
-  .reservation-list {
-    grid-template-columns: 1fr;
+  .reservation-page {
+    width: 100%;
+  }
+
+  .swiper-button-prev,
+  .swiper-button-next {
+    display: none;
+  }
+}
```

Comment:
In the mobile version, you have all of your items in one row. However, that is not a good user experience. Everyone is accustomed to scrolling down when they have images like this. 
So, make sure you meet your users' needs by showing your items `vertically` not `horizontally`

Reply:
hi @Muhammad0602 ,
thanks for your feedback,
I am not sure but, in the design, as I got it is horizontally and with slider kindly check the link once and let me know if I am worng
![mobil responsiv](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend/assets/108926875/a5f30a53-a7c5-44c5-a5f8-c994b8d5573e)

https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign

Reply:
This is the main page. But ok, you can keep it as it is. It looks nice.